content/guides/userguides/terra-export.md

```diff
@@ -0,0 +1,126 @@
+---
+path: "/guides/userguides/consumer-vignettes/export-to-terra"
+date: "2019-09-17"
+title: "Exporting HCA Data to Terra"
+subTitle: ""
+---
+
+Exporting search results from the HCA Data Explorer to Terra
+============================================================
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
+In particular, you'll want to be acquainted with the content in these
+tutorials:
+
+-   [Accessing HCA Data and Metadata][quick-start-guide]
+
+You'll want to use Google Chrome for this tutorial; Terra has poorer support
```

Comment:
How is this expressed on the terra docs? Can we say terra recommends you use google chrome if that is actually the case?

Reply:
From their "How to register for a Terra account" howto (https://support.terra.bio/hc/en-us/articles/360028235911-How-to-register-for-a-Terra-account):

> NOTE: Terra is designed and tested (and will run best on) on the Chrome browser. If you are having problems and are using a different browser, try switching to Chrome and see if that resolves the issue.

Reply:
Fixed working around recommended browser and added the link above